libazureinit/src/media.rs

```diff
@@ -406,4 +418,84 @@ mod tests {
             _ => panic!("Non-empty passwords aren't allowed"),
         };
     }
+
+    #[test]
+    fn test_get_mount_device_with_cdrom_entries() {
+        let mut temp_file =
+            NamedTempFile::new().expect("Failed to create temporary file");
+        let mount_table = r#"
+            /dev/sr0 /mnt/cdrom iso9660 ro,user,noauto 0 0
+            /dev/sr1 /mnt/cdrom2 udf ro,user,noauto 0 0
+        "#;
+        temp_file
+            .write_all(mount_table.as_bytes())
+            .expect("Failed to write to temporary file");
+        let temp_path = temp_file.into_temp_path();
+        let result = get_mount_device(Some(temp_path.as_ref()));
+
+        assert!(result.is_ok());
```

Comment:
The test fails if it panics so you could leave this out since the next line unwraps the result.

Reply:
Removed this statement from all the tests. Thank you!